scripts/jsonschema2protobuf.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
```

Comment:
Where does this script comes from? What is the copyright? Author?
A toplevel comment explaining those things, limitations, etc. would be useful.

Reply:
Added